### DISCUSSION TITLE
Is it only possibly to signin via Javascript?

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#980] [Q&A]
I've somehow managed to never develop with JWT signin before.  Does Pocketbase require either `authViaEmail` or `authViaEmail` to signin?  I was digging around for a normal signin and I couldn't find it.

**Comment by ganigeorgiev** [comment]
What do you mean by a "normal signin"? PocketBase uses JWT tokens via the `Authorization` HTTP header and JavaScript is not required:

```
// for v0.7.x
Authorization: User TOKEN
Authorization: Admin TOKEN

// for v0.8.0-rc
Authorization: TOKEN // for both user and admin authentication
```

Please note that PocketBase API is stateless, aka. there are no "sessions" and you need to send the Authorization token with each request.
